pages/about/policies.html

```diff
@@ -180,7 +186,7 @@ <h1 class="ui header">Secondary School Literacy</h1>
             </div>
           </div>
         </div><!-- End Row 5 -->
-        <!-- Start Row 5 -->
+        <!-- Start Row 6 -->
```

Comment:
@evanhearne same again here, this was added during the pr we worked through last night